ioc_rix_sp1k1_calc/ioc_rix_sp1k1_calc.py

```diff
@@ -188,3 +252,19 @@ def calculate_grating(self) -> str:
         if self.g_h_value is None:
             return ""
         return rix_calibration.get_grating()
+
+    async def _update_focaldist_calc(self, timestamp):
+        """
+        Update our focal distance PVs based on the most recent values.
+        """
+        new_mr3k2_focaldist, new_mr4k2_focaldist = self.calculate_focaldists()
```

Comment:
I checked `/cds/group/pcds/dist/pds/rix/scripts/rix_calibration.py` to confirm that this is the correct ordering (mr3k2 first, then mr4k2) :+1: